src/api/java/com/minecolonies/api/colony/requestsystem/management/IRequestHandler.java

```diff
@@ -148,4 +148,16 @@
      * @param requester the requester.
      */
     void removeRequester(IRequester requester);
+
+    /**
+     * Add a resolver to the temporary blacklist.
+     * @param resolverToken the resolver uuid.
+     */
+    void addToTempBlackList(IToken<?> resolverToken);
+
+    /**
+     * Remove a resolver from the temporary blacklist.
+     * @param resolverToken the resolver uuid.
+     */
+    void removeFromTempBlackList(IToken<?> resolverToken);
```

Comment:
This needs to be in the resolver handler.
This has intrinsically more todo with resolvers and not so much with requests.